packages/material-ui/src/Box/styleFunction.test.js

```diff
@@ -213,5 +213,27 @@ describe('styleFunction', () => {
         '@media (min-width:960px)': { padding: '70px', margin: '30px' },
       });
     });
+
+    it('writes breakpoints in correct order', () => {
+      const result = styleFunction({
+        theme,
+        m: { md: 1, lg: 2 },
+        sx: { p: { xs: 0, sm: 1, md: 2 } },
```

Comment:
Would it be better for the test case to start from the sx prop?

```suggestion
        sx: { m: { md: 1, lg: 2 }, p: { xs: 0, sm: 1, md: 2 } },
```

Reply:
Will add additional test for it 👍